apps/Launcher/ModelFilter.cpp

```diff
@@ -0,0 +1,58 @@
+/*********************************************************************/
+/* Copyright (c) 2018, EPFL/Blue Brain Project                       */
+/*                     Raphael Dumusc <raphael.dumusc@epfl.ch>       */
+/* All rights reserved.                                              */
+/*                                                                   */
+/* Redistribution and use in source and binary forms, with or        */
+/* without modification, are permitted provided that the following   */
+/* conditions are met:                                               */
+/*                                                                   */
+/*   1. Redistributions of source code must retain the above         */
+/*      copyright notice, this list of conditions and the following  */
+/*      disclaimer.                                                  */
+/*                                                                   */
+/*   2. Redistributions in binary form must reproduce the above      */
+/*      copyright notice, this list of conditions and the following  */
+/*      disclaimer in the documentation and/or other materials       */
+/*      provided with the distribution.                              */
+/*                                                                   */
+/*    THIS  SOFTWARE IS PROVIDED  BY THE  UNIVERSITY OF  TEXAS AT    */
+/*    AUSTIN  ``AS IS''  AND ANY  EXPRESS OR  IMPLIED WARRANTIES,    */
```

Comment:
Hmm proabably not, I just keep copying the same disclaimer everywhere. We use "Ecole polytechnique federale de Lausanne" at the end of it so it's not even consistent. Since we are the ones providing the software now a batch change of all files would be a good idea, keeping only the "Copyright (c) 2011-2012, The University of Texas at Austin" where applicable (at the top of old files).